packages/app/src/domain/vaccine/vaccine-coverage-per-age-group-vr-gm.tsx

```diff
@@ -0,0 +1,225 @@
+import {
+  GmVaccineCoveragePerAgeGroupValue,
+  VrVaccineCoveragePerAgeGroupValue,
+} from '@corona-dashboard/common';
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import { isPresent } from 'ts-is-present';
+import { Box } from '~/components/base';
+import { ChartTile } from '~/components/chart-tile';
+import { Markdown } from '~/components/markdown';
+import { InlineText } from '~/components/typography';
+import { useIntl } from '~/intl';
+import { asResponsiveArray } from '~/style/utils';
+import { assert } from '~/utils/assert';
+import { replaceVariablesInText } from '~/utils/replace-variables-in-text';
+import { parseBirthyearRange } from './logic/parse-birthyear-range';
+import { parseFullyVaccinatedPercentageLabel } from './logic/parse-fully-vaccinated-percentage-label';
+interface VaccineCoveragePerAgeGroupVrGmProps {
+  title: string;
+  description: string;
+  annotation_description?: string;
+  topLabels: {
+    agegroups: string;
+    vaccination_coverage: string;
+    with_1_shot: string;
```

Comment:
```suggestion
    with_one_shot: string;
```

Clearer names always win over shorter names.